### DISCUSSION TITLE
Modeling "Paper" XML

### DISCUSSION THREAD

**Discussion by aryakeyvan-star** [google-deepmind/mujoco#2129] [Asking for Help]
### Intro

Hello, I am student and using Mujoco for robotic applications.

### My setup

Mujoco v.323

### My question

Hello,

I am trying to model a sheet of paper in mujoco in the mujoco xml format. Currently I am using the flexcomp class and the first party plugin: mujoco.elasticity.shell . I have also tried the composite class and end up with the same issues. The issue is that when I simulate, the paper performs more like cloth. I cannot make it stiffer without running into issues. Any help would be appreciated. I have attached my xml...    

### Minimal model and/or code that explain my question

Paper XML Model:
<details>
<summary>minimal XML</summary>

```XML

<mujoco model="Paper">

  <extension>
    <plugin plugin="mujoco.elasticity.shell"/>
  </extension>

  <worldbody>
    <flexcomp type="grid" count="10 16 1" spacing="0.013 0.013 0.013" radius="0.0001" name="paper" mass="5e-3">
      <edge equality="true" damping="0.001"/>
      <plugin plugin="mujoco.elasticity.shell">
        <config key="poisson" value="0.3"/>
        <config key="young" value="3e9"/>
        <config key="thickness" value="1e-5"/>
      </plugin>
     <pin id="0"/>
     <pin id="15"/>
    </flexcomp>     
  </worldbody>    

  

</mujoco>       

```

### Confirmations

- [X] I searched the [latest documentation](https://mujoco.readthedocs.io/en/latest/overview.html) thoroughly before posting.
- [X] I searched previous [Issues](https://github.com/google-deepmind/mujoco/issues) and [Discussions](https://github.com/google-deepmind/mujoco/discussions), I am certain this has not been raised before.

**Comment by quagla** [comment]
What issues are you referring to?

**Reply by aryakeyvan-star** [reply]
I have written my issue in my original post and below. Let me know if there is any other clarification needed.

**Comment by aryakeyvan-star** [comment]
The issue is that I cannot figure out what properties to change in the flexcomp to get the model to act more like paper compared to cloth. Any ideas would be appreciated. My XML is attached my original post above. Let me know if there is any other clarification needed.
